src/guides/v2.3/frontend-dev-guide/css-topics/css-overview.md

```diff
@@ -58,6 +58,20 @@ ExampleCorp wants to change the color of the primary buttons to orange. To achie
    @button-primary__hover__border: 1px solid @color-orange-red2;
    ```
 
+ExampleCorp wants to change the color of the primary buttons to a custom color. In the Orange theme directory add the overriding `app/design/frontend/ExampleCorp/orange/web/css/source/_theme.less` file with the following code:
```

Comment:
Hi, all!

This sentense may confuse a reader.

There is a defference between `ExampleCorp/orange` theme and `Orange` theme. Please replace  `Orange` with `orange`.

Thank you!

Reply:
Hi @atwixfirster,
Thanks for your review and suggestion. Requested suggestion updated.

Reply:
Hi, rajesh-mage !

Looks better for me now.

P.S. Please sign the CLA. Without that your PR will be not accepted and merged.

Thank you!

Reply:
Hi, @atwixfirster
Adobe CLA sign creation is already done.